### PR TITLE
Add: `Flywheel.Local` version 6.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
#21070 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21073)

### CODE REVIEW

File: manifests/f/Flywheel/Local/6.0.0/Flywheel.Local.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+LicenseUrl: https://localwp.com/legal
+Copyright: Copyright (c) 2021 Local
+CopyrightUrl: https://localwp.com/legal
+ShortDescription: Local is a free local development environment designed to simplify the workflow of WordPress developers and designers. Download it for free today!
+#Description: 
+Moniker: flywheel-local
+Tags:
+- wordpress sites
+- flywheel
+- web development
```
Comment (ItzLevvie) [inline manifests/f/Flywheel/Local/6.0.0/Flywheel.Local.locale.en-US.yaml:24]:
```suggestion
- wordpress
- sites
- flywheel
- web
- development
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/f/Flywheel/Local/6.0.0/Flywheel.Local.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 6.0.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
```
Comment (ItzLevvie) [inline manifests/f/Flywheel/Local/6.0.0/Flywheel.Local.installer.yaml:14]:
```suggestion
  Scope: user
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing https://github.com/microsoft/winget-pkgs/pull/21073/files/9ce44cd5c5683bb0a31bd1e288d6bff304b01799#r670514697.

You can either put it in a new line or replace spaces with `-`.

File: manifests/f/Flywheel/Local/6.0.0/Flywheel.Local.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://cdn.localwp.com/releases-stable/6.0.0+5444/local-6.0.0-windows.exe
+  InstallerSha256: 9FE7DAF99B22C91B53785340F47678EF450B2DC485352CEBFBA9B51537EB1F6E
+  UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/f/Flywheel/Local/6.0.0/Flywheel.Local.installer.yaml:17]:
```suggestion
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://cdn.localwp.com/releases-stable/6.0.0+5444/local-6.0.0-windows.exe
  InstallerSha256: 9FE7DAF99B22C91B53785340F47678EF450B2DC485352CEBFBA9B51537EB1F6E
  InstallerSwitches:
    Custom: /CURRENTUSER
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://cdn.localwp.com/releases-stable/6.0.0+5444/local-6.0.0-windows.exe
  InstallerSha256: 9FE7DAF99B22C91B53785340F47678EF450B2DC485352CEBFBA9B51537EB1F6E
  InstallerSwitches:
    Custom: /ALLUSERS
  UpgradeBehavior: install
```
Needed for compatibility

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**